static/web.js

```diff
@@ -553,6 +575,8 @@
         themes = document.getElementById("themes");
         editor = ace.edit("editor");
         set_result.editor = editor;
+        editor.$blockScrolling = Infinity;
```

Comment:
[Current](https://github.com/rust-lang/rust-playpen/commit/88045c4b28140d2dd6afeb34c5ef6174627a46ff) rust-playpen actually gives the warning twice:
![blockcursor](https://cloud.githubusercontent.com/assets/17984080/14235237/7cef2f94-f9f8-11e5-9de8-8742a897f200.png)